types/chrome/test/index.ts

```diff
@@ -754,36 +754,45 @@ function testWindows() {
 
 // https://developer.chrome.com/extensions/storage#type-StorageArea
 function testStorage() {
-    function getCallback(loadedData: { [key: string]: any }) {
-        var myValue: { x: number } = loadedData["myKey"];
+    interface StorageData {
+        myKey: {
+            x: number;
+            y: number;
+            z?: number;
+        };
+        myKey2: string;
+    }
+
+    function getCallback(loadedData: StorageData) {
+        console.log(loadedData.myKey.x + loadedData.myKey.y);
     }
 
-    chrome.storage.sync.get(getCallback);
-    chrome.storage.sync.get("myKey", getCallback);
-    chrome.storage.sync.get(["myKey", "myKey2"], getCallback);
-    chrome.storage.sync.get({ foo: 1, bar: 2 }, getCallback);
-    chrome.storage.sync.get(null, getCallback);
+    chrome.storage.sync.get<StorageData>(getCallback);
```

Comment:
Can we keep a version of all these tests with and without setting the generic type? (e.g. so it uses the default, and we're more confident we haven't broken existing code, and won't in future)